BuildTools/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL "Linux")
     set(CLANG_FORMAT_EXECUTABLE "${CMAKE_CURRENT_SOURCE_DIR}/FormatValidation/clang-format_linux_10.0.0" CACHE INTERNAL "clang-format executable path")
 elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL "Darwin")
     set(CLANG_FORMAT_EXECUTABLE "${CMAKE_CURRENT_SOURCE_DIR}/FormatValidation/clang-format_mac_10.0.0" CACHE INTERNAL "clang-format executable path")
+elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL "FreeBSD")
+    set(CLANG_FORMAT_EXECUTABLE "/usr/local/llvm10/bin/clang-format" CACHE INTERNAL "clang-format executable path")
```

Comment:
What happens if there is no llvm10 installed? There will be an error that is hard to understand.

I am thinking about disabling format validation completely - it is intended for development, but is not required for build.

Reply:
This part here is specifically for the format validation.  This part here (along with the scripts I added) may be omitted entirely if desired.